.github/workflows/release-series-images.yml

```diff
@@ -95,33 +80,31 @@ jobs:
         # which is required for build-push-action
         IFS=,
         echo "::set-output name=taglist::${tag_list[*]}"
-
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
-
-    - name: Log in to Docker Hub
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
       with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-
-    - name: Log in to OSG Harbor
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
+        driver: docker
```

Comment:
Huh, is this necessary?

Reply:
So I think from what I found my understanding is that it has to do with the load:true setup. The gha wasn't working for me when I did not set the driver to docker because I think has the default driver set as docker-container.

Reply:
Could you add a comment to note that?